default.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Default preset for use with OpenFeature's repos",
+  "extends": [
+    "config:base",
+    "prConcurrentLimit10"
+  ],
+  "automergeType": "branch",
+  "configMigration": true,
+  "dependencyDashboard": true,
+  "prCreation": "not-pending",
+  "rebaseWhen": "behind-base-branch",
+  "platformAutomerge": true,
```

Comment:
Are we confident enough in the test coverage across repos to include this in the base config?

Reply:
Yeah, good catch. I think it's best to remove this (default is disabled).

https://docs.renovatebot.com/configuration-options/#platformautomerge